model/task/task.go

```diff
@@ -669,6 +669,15 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 
 	t.TimeTaken = finishTime.Sub(t.StartTime)
 	t.Details = *detail
+
+	grip.Info(message.Fields{
+		"message":   "marking task finished",
```

Comment:
does this capture which host it ran on?

Reply:
I don't think it needs to. This is for logging setup-failed tasks. We're only interested in the task info.